pkg/clients/ovirt/impl.go

```diff
@@ -217,7 +223,7 @@ func (is *ovirtClient) DeleteVM(id string) error {
 	vmService := is.connection.SystemService().VmsService().VmService(id)
 	_, err := vmService.Stop().Send()
 	if err != nil {
-		return err
+		return fmt.Errorf("error stopping VM %s: %w", id, err)
```

Comment:
can we use error.Wrapf instead ?

Reply:
I think error.Wrapf shouldn't be used any more once Errorf and %w was introduced to golang 1.13.
Plus it ignores nil errors which I personaly find annoying :)